addons/survey/models/survey_question.py

```diff
@@ -265,6 +265,17 @@ def _compute_is_scored_question(self):
             else:
                 question.is_scored_question = False
 
+    # ------------------------------------------------------------
+    # CRUD
+    # ------------------------------------------------------------
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_prevention_of_survey_question(self):
+        """prevent deletion of question when the session is in progress"""
+        for survey_id in self.survey_id:
+            if survey_id.session_state == 'in_progress':
+                raise UserError(_('You cannot delete a question from the survey "%s" while the live session is in progress.', survey_id.title))
```

Comment:
We can also help translators with something like this:

```suggestion
                raise UserError(_(
                    'You cannot delete a question from the survey "%(title)s" while the live session is in progress.',
                    title=survey_id.title,
                ))
```